components/motor/roboclaw/roboclaw.go

```diff
@@ -56,14 +56,14 @@ const (
 type Config struct {
 	SerialPath       string `json:"serial_path"`
 	SerialBaud       int    `json:"serial_baud_rate"`
-	Number           int    `json:"number_of_motors"` // this is 1 or 2
+	Channel          int    `json:"motor_channel"` // this is 1 or 2
 	Address          int    `json:"address,omitempty"`
 	TicksPerRotation int    `json:"ticks_per_rotation,omitempty"`
 }
 
 // Validate ensures all parts of the config are valid.
 func (conf *Config) Validate(path string) ([]string, error) {
-	if conf.Number < 1 || conf.Number > 2 {
+	if conf.Channel < 1 || conf.Channel > 2 {
 		return nil, conf.wrongNumberError()
```

Comment:
nit: maybe consider changing `wrongNumberError()` to `wrongChannelError()` for consistency?